plugins/modules/pacemaker_cluster.py

```diff
@@ -26,18 +27,20 @@
   state:
     description:
       - Indicate desired state of the cluster.
-    choices: [cleanup, offline, online, restart]
+      - The value V(maintenance) has been added in community.general 11.1.0.
+    choices: [cleanup, offline, online, restart, maintenance]
     type: str
-  node:
+  name:
     description:
       - Specify which node of the cluster you want to manage. V(null) == the cluster status itself, V(all) == check the status
         of all nodes.
     type: str
+    aliases: ['node']
   timeout:
     description:
-      - Timeout when the module should considered that the action has failed.
-    default: 300
+      - Timeout period for polling the cluster operation.
```

Comment:
This is in seconds, I guess?
```suggestion
      - Timeout period (in seconds) for polling the cluster operation.
```